build.gradle.kts

```diff
@@ -0,0 +1,136 @@
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+
+plugins {
+    kotlin("jvm") version "1.7.0"
+    kotlin("plugin.allopen") version "1.7.0"
+    id("io.quarkus")
+    id("org.openapi.generator") version "5.4.0"
+    id("org.jetbrains.kotlin.kapt") version "1.7.0"
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+val quarkusPlatformGroupId: String by project
+val quarkusPlatformArtifactId: String by project
+val quarkusPlatformVersion: String by project
+val jaxrsFunctionalTestBuilderVersion: String by project
+val testContainersVersion: String by project
+val testContainersKeycloakVersion: String by project
+val moshiVersion: String by project
+val wiremockVersion: String by project
+
+dependencies {
+    implementation(enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}"))
+    implementation("io.quarkus:quarkus-hibernate-orm")
+    implementation("io.quarkus:quarkus-container-image-docker")
+    implementation("io.quarkus:quarkus-hibernate-validator")
+    implementation("io.quarkus:quarkus-kotlin")
+    implementation("io.quarkus:quarkus-oidc")
+    implementation("io.quarkus:quarkus-keycloak-admin-client")
+    implementation("io.quarkus:quarkus-resteasy-jackson")
+    implementation("io.quarkus:quarkus-liquibase")
+    implementation("io.quarkus:quarkus-jdbc-mysql")
+    implementation("io.quarkus:quarkus-scheduler")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("io.quarkus:quarkus-arc")
```

Comment:
If possible then would probably organise the order of the modules as starts with io.quarkus then jumps to other module types like jetbrains and jumps back to io.quarkus.

Reply:
fixed